telemetry/opentelemetry-config/src/main/java/io/helidon/telemetry/otelconfig/SamplerConfigBlueprint.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.telemetry.otelconfig;
+
+import java.util.Optional;
+
+import io.helidon.builder.api.Option;
+import io.helidon.builder.api.Prototype;
+
+/**
+ * Tracing sampler settings.
+ */
+@Prototype.Blueprint
+@Prototype.Configured
+@Prototype.CustomMethods(SamplerConfigSupport.CustomMethods.class)
+interface SamplerConfigBlueprint {
+
+    /**
+     * Sampler type.
+     *
+     * @return sampler type
+     */
+    @Option.Configured
+    @Option.Default(SamplerType.DEFAULT_NAME)
+    SamplerType type();
+
+    /**
+     * Sampler parameter.
+     *
+     * @return sampler parameter
+     */
+    @Option.Configured
+    Optional<Number> param();
```

Comment:
This will most likely not work - either change to `Optional<Double>`, or you should add a factory method.